bcda/models/service_test.go

```diff
@@ -310,6 +395,132 @@ func (s *ServiceTestSuite) TestGetBeneficiaries() {
 	}
 }
 
+func (s *ServiceTestSuite) TestGetQueJobs() {
+
+	defaultACOID, priorityACOID := "SOME_ACO_ID", "PRIORITY_ACO_ID"
+	os.Setenv("PRIORITY_ACO_IDS", priorityACOID)
+
+	benes1, benes2 := make([]*CCLFBeneficiary, 10), make([]*CCLFBeneficiary, 20)
+	allBenes := [][]*CCLFBeneficiary{benes1, benes2}
+	for idx, b := range allBenes {
+		for i := 0; i < len(b); i++ {
+			id := uint(idx*10000 + i + 1)
+			b[i] = getCCLFBeneficiary(id, fmt.Sprintf("MBI%d", id))
+		}
+	}
+	benes1MBI := make([]string, 0, len(benes1))
+	benes1ID := make(map[string]struct{})
+	for _, bene := range benes1 {
+		benes1MBI = append(benes1MBI, bene.MBI)
+		benes1ID[strconv.FormatUint(uint64(bene.ID), 10)] = struct{}{}
+	}
+
+	since := time.Now()
+	serviceDate := time.Date(since.Year()-1, 12, 31, 23, 59, 59, 999999, time.UTC)
+
+	type test struct {
+		name           string
+		acoID          string
+		reqType        RequestType
+		expSince       time.Time
+		expServiceDate time.Time
+		expBenes       []*CCLFBeneficiary
+		resourceTypes  []string
+	}
+
+	// TODO - Figure out job priority
```

Comment:
Is this a separate ticket?

Reply:
Ah, stale! I added tests verifying priority calculations.